mvrx/src/test/kotlin/com/airbnb/mvrx/MvRxViewModelProviderTest.kt

```diff
@@ -85,6 +85,12 @@ class MvRxViewModelProviderTest : BaseTest() {
         }
     }
 
+    @Test(expected = IllegalStateException::class)
+    fun failForAccessingViewModelBeforeOnCreate() {
+        val activityWithoutSetup = Robolectric.buildActivity(FragmentActivity::class.java).get()
```

Comment:
ideally this would test the error message too, since we know this will  fail for some other reason at least, but we want to make sure it fails with your custom error message